corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js

```diff
@@ -251,6 +252,19 @@ WebFormSession.prototype.handleFailure = function (resp, action, textStatus, fai
     } else if (resp.hasOwnProperty('responseJSON') && resp.responseJSON !== undefined) {
         errorMessage = Formplayer.Utils.touchformsError(resp.responseJSON.message);
     }
+
+    hqImport('cloudcare/js/util').reportFormplayerErrorToHQ({
+        type: 'webformsession_request_failure',
+        request: action,
+        readableErrorMessage: errorMessage,
```

Comment:
Hey Shivam, this is a good question and something I'm not 100% sure about. But when I was working on this (and I double checked just now), I found that a "success" response (200 response) could not cause the spinner next to the answer to turn into a ❌or make the error text show up beneath the answer. It can show a red banner error, but I'm already logging those separately from this, and those are showing up there.